src/main/scala/cromwell/engine/backend/local/LocalBackendCall.scala

```diff
@@ -35,4 +36,8 @@ case class LocalBackendCall(backend: LocalBackend,
 
   override def useCachedCall(avoidedTo: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] =
     backend.useCachedCall(avoidedTo.asInstanceOf[LocalBackendCall], this)
+
+  override def pollingInitialInterval: FiniteDuration = 10.seconds
+  override def pollingMaxInterval: FiniteDuration = 10.minutes
+  override def pollingMultiplier: Double = 1.1D
```

Comment:
There is technically 1 "~polling" happening. Looking at the code when the `execute` Future returns its `ExecutionHandle`, the `CallExecutionActor` calls `poll`, which in the case of `LocalBackend` and `SgeBackend` returns immediately the same handle. Nonetheless the `CallExecutionActor` uses the `ExponentialBackend` object once to do this.

But let me see if I can refactor those `pollingInterval`s in a cleaner way.